README.md

```diff
@@ -1,86 +1,173 @@
+
+▶ kubectl get po
+NAME                   READY     STATUS    RESTARTS   AGE
+helloapache            1/1       Running   0          2m
+k8s-etcd-127.0.0.1     1/1       Running   0          1d
+k8s-master-127.0.0.1   4/4       Running   0          1d
+k8s-proxy-127.0.0.1    1/1       Running   0          1d
 ```
-docker build -t [TAG] .
+
+__Fetch, edit and run Apache on Kubernetes:__
+```sh
+▶ sudo atomicapp fetch --destination helloapache projectatomic/helloapache
+2016-02-25 15:35:41,439 - [INFO] - main.py - Action/Mode Selected is: fetch
+2016-02-25 15:35:41,440 - [INFO] - base.py - Unpacking image: projectatomic/helloapache to helloapache
+2016-02-25 15:35:45,067 - [INFO] - container.py - Skipping pulling Docker image: projectatomic/helloapache
+2016-02-25 15:35:45,067 - [INFO] - container.py - Extracting nulecule data from image: projectatomic/helloapache to helloapache
+c12d2047fab44f2906b9cbee3ac86c6c6499921ce33a90085e8765491b44f447
+
+Your application resides in helloapache
+Please use this directory for managing your application
+
+▶ cd helloapache 
+
+▶ cat Nulecule
+...
+      - name: hostport
+        description: The host TCP port as the external endpoint
+        default: 80
+...
+
+▶ vim Nulecule 
+▶ cat Nulecule 
+...
+      - name: hostport
+        description: The host TCP port as the external endpoint
+        default: 8080
+...
+
+▶ sudo atomicapp run --provider=kubernetes .
+...
+
+▶ # or
+
+▶ docker build -t myapp
+...
+▶ sudo atomicapp run --provider=kubernetes myapp
+...
 ```
 
-Use 'docker build' to package up the application and tag the resulting image.
+## Real-world examples
+Atomic App can be used to launch a cluster of containers (application servers, databases, etc.).
 
-### Fetch and run
-```
-atomicapp [--dry-run] [-v] [-a answers.conf] fetch|run|stop|genanswers [--provider docker] [--destination DST_PATH] APP|PATH
-```
+For a list of already packaged examples, visit the [nulecule-library](https://github.com/projectatomic/nulecule-library) repo.
 
-Pulls the application and its dependencies. If the last argument is
-existing path, it looks for `Nulecule` file there instead of pulling anything.
+## Contributing to Atomic App
+[![Code Health](https://landscape.io/github/projectatomic/atomicapp/master/landscape.svg?style=flat)](https://landscape.io/github/projectatomic/atomicapp/master)
+[![Build Status](https://travis-ci.org/projectatomic/atomicapp.svg?branch=master)](https://travis-ci.org/projectatomic/atomicapp)
+[![Coverage Status](https://coveralls.io/repos/projectatomic/atomicapp/badge.svg?branch=master&service=github)](https://coveralls.io/github/projectatomic/atomicapp?branch=master)
+[![Issue Stats](http://issuestats.com/github/projectatomic/atomicapp/badge/pr)](http://issuestats.com/github/projectatomic/atomicapp)
+[![Issue Stats](http://issuestats.com/github/projectatomic/atomicapp/badge/issue)](http://issuestats.com/github/projectatomic/atomicapp)
```

Comment:
ok, that's fine. I just assumed you had mistakenly put it there. I usually see them at the top.